core/src/test/java/com/sap/ai/sdk/core/client/DeploymentUnitTest.java

```diff
@@ -31,7 +33,7 @@
  */
 public class DeploymentUnitTest extends WireMockTestServer {
   @Test
-  void testGetDeployments() {
+  void getDeployments() {
```

Comment:
(Question)

Just wondering, why change the test method name?

Reply:
Because when a test fails, the logs will read:
`DeploymentUnitTest.getDeployments()`

Instead of:
`DeploymentUnitTest.testgetDeployments()`

It's more readable

Reply:
I understand and approve of the sentiment.

Be aware that your naming strategy may fall apart, when you go with more than 1 test per public method.
E,g,
```
DeploymentUnitTest.testgetDeployments()
DeploymentUnitTest.testgetDeploymentsWithMultiTenancy()
DeploymentUnitTest.testgetDeploymentsWithResilience()
DeploymentUnitTest.testgetDeploymentsWithErrors()
```

Maybe it'll be fine anyway.